rqlite/post_install.md

```diff
@@ -0,0 +1,6 @@
+## Rqlite - The lightweight, distributed relational database built on SQLite.
```

Comment:
I always spell it with a lowercase 'r', FWIW.  As in "rqlite - the lightweight, distributed relational database built on SQLite."

Reply:
Thanks Philip for the extra 👀 I updated this and validated lowercase everywhere else too. While you're here I just grabbed the logo.png from the site not sure if you have a different image you want in. It should work but if we want to fill it out the space more we can max up to 512 x 512. More details here on the image sizing if you think we should tweak it at all.
- https://github.com/civo/kubernetes-marketplace#writing

Reply:
No problem using the logo, go for it.